tests/queries/0_stateless/03442_string_bytes_functions.sql

```diff
@@ -0,0 +1,18 @@
+SELECT stringBytesUniq('Hello');
+SELECT stringBytesUniq('');
+SELECT stringBytesUniq('aaaa');
+SELECT stringBytesUniq('abcABC123');
+SELECT stringBytesUniq(toNullable('Hello'));
+SELECT stringBytesUniq(toLowCardinality('Hello'));
+
+SELECT stringBytesEntropy('Hello');
+SELECT stringBytesEntropy('');
+SELECT stringBytesEntropy('aaaa');
+SELECT stringBytesEntropy('abcABC123');
+SELECT stringBytesEntropy(toNullable('Hello'));
+SELECT stringBytesEntropy(toLowCardinality('Hello'));
+
```

Comment:
At least one test when the string is non constant (there is more than one string in a block).

Reply:
Added.